hack/local-up-cluster.sh

```diff
@@ -947,6 +947,15 @@ EOF
 fi
 }
 
+# If we are running in the CI, we need a few more things before we can start
+if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
```

Comment:
KUBETEST_IN_DOCKER does not say what it is. Maybe RUN_IN_KUBE_CI ?

Reply:
@sttts KUBETEST_IN_DOCKER is already used by the CI system. please see https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-local-e2e/219/build-log.txt